infra-as-code/bicep/ai-foundry-project.bicep

```diff
@@ -202,6 +187,7 @@ resource aiFoundry 'Microsoft.CognitiveServices/accounts@2025-06-01' existing  =
       }
       dependsOn: [
         applicationInsightsConnection  // Single thread changes to the project, else conflict errors tend to happen
+        projectBlobDataOwnerConditionalAssignment
```

Comment:
Isn't this already handled in the chain, on `storageConnection`?

Reply:
yes it is @ckittel, so we removed the redundant dependency

done | addressed from 13e8d6a04a8803daafdf007ca68a68a9949bdde9